api/scpca_portal/migrations/0034_auto_20240131_2320.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.22 on 2024-01-31 23:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("scpca_portal", "0033_computedfile_modality"),
```

Comment:
```suggestion
        ("scpca_portal", "0034_auto_20240131_1642"),
```

Reply:
Django will take care of that -- they will be combined just fine (there will be an extra migration for that).